source/integrate-with-integration-environment/process-identity-information.html.md.erb

```diff
@@ -0,0 +1,295 @@
+
+Our support team will give you the public key you must use to verify this JWT. We recommend you [use a certified JWT/JWS implementation](https://openid.net/developers/jwt/).
+
+The JWT contains the following claims:
+
+```
+{
+  "sub": "urn:fdc:gov.uk:2022:56P4CMsGh_02YOlWpd8PAOI-2sVlB2nsNU7mcLZYhYw=",
+  "iss": "https://identity.integration.account.gov.uk/",
+  "nbf": 1541493724,
+  "iat": 1541493724,
+  "exp": 1573029723,
+  "vot": "P2",
+  "vtm": "https://oidc.integration.account.gov.uk/trustmark",
+  "vc": {
+    "type": [
+      "VerifiableCredential",
+      "VerifiableIdentityCredential"
+    ],
+    "credentialSubject": {
+      "name": [
+        {
+          "validFrom": "2020-03-01",
+          "nameParts": [
+            {
+              "value": "Alice",
+              "type": "GivenName"
+            },
+            {
+              "value": "Jane",
+              "type": "GivenName"
+            },
+            {
+              "value": "Laura",
+              "type": "GivenName"
+            },
+            {
+              "value": "Doe",
+              "type": "FamilyName"
+            }
+          ]
+        },
+        {
+          "validUntil": "2020-03-01",
+          "nameParts": [
+            {
+              "value": "Alice",
+              "type": "GivenName"
+            },
+            {
+              "value": "Jane",
+              "type": "GivenName"
+            },
+            {
+              "value": "Laura",
+              "type": "GivenName"
+            },
+            {
+              "value": "O’Donnell",
+              "type": "FamilyName"
+            }
+          ]
+        }
+      ],
+      "birthDate": [
+        {
+          "value": "1970-01-01"
+        }
+      ]
+    }
+  }
+}
+```
+
+The `vc` claim in the JWT is a [verifiable credential (VC)](https://www.w3.org/TR/vc-data-model/). Claims about your user are contained in the `credentialSubject` JSON object.
+
+### Validate your user’s identity credential
+
+1. You must validate the JWT signature according to the [JSON Web Signature Specification](https://datatracker.ietf.org/doc/html/rfc7515). Check that the JWT `alg` header is `ES256`. Then you can use the value of the JWT `alg` header parameter to validate the JWT.
+1. Check the `iss` claim is `https://identity.integration.account.gov.uk/`.
+1. Check the `sub` claim matches the `sub` claim you received in [the `id_token` from your token request](../integrate-with-code-flow/#understand-your-id-token).
+1. Check the current time is before the time in the `exp` claim.
+
+### Check your user’s identity credential matches the level of confidence needed
+
+You must check the `vot` claim in the JWT matches or exceeds the level of confidence you need to let your user access your service. You may receive a lower level of confidence than you requested. For example, if you asked for medium confidence (`P2`), you may receive an identity credential containing low confidence (`P1` in the `vot` claim).
```

Comment:
```suggestion
You must check the `vot` claim in the JWT matches or exceeds the level of confidence you need to let your user access your service. 

You may receive a lower level of confidence than you requested. For example, if you asked for medium confidence (`P2`), you may receive an identity credential containing low confidence (`P1` in the `vot` claim).
```

Reply:
I'm not sure about this as the first and second sentences should go together - they're about the same thing. We could make that link more clear by changing the beginning of the second sentence slightly...

```suggestion
You must check the `vot` claim in the JWT matches or exceeds the level of confidence you need to let your user access your service. This is because you may receive a lower level of confidence than you requested. For example, if you asked for medium confidence (`P2`), you may receive an identity credential containing low confidence (`P1` in the `vot` claim).
```